src/Transpiler.ts

```diff
@@ -342,50 +347,8 @@ export abstract class LuaTranspiler {
     }
 
     public transpileImport(node: ts.ImportDeclaration): string {
-        const importPath = this.transpileExpression(node.moduleSpecifier);
-        const importPathWithoutQuotes = importPath.replace(new RegExp("\"", "g"), "");
-
-        if (!node.importClause || !node.importClause.namedBindings) {
-            throw TSTLErrors.DefaultImportsNotSupported(node);
-        }
-
-        const imports = node.importClause.namedBindings;
-
-        const requireKeyword = "require";
-
-        if (ts.isNamedImports(imports)) {
-            const fileImportTable = path.basename(importPathWithoutQuotes) + this.importCount;
-            const resolvedImportPath = this.getImportPath(importPathWithoutQuotes);
-
-            let result = `local ${fileImportTable} = ${requireKeyword}(${resolvedImportPath})\n`;
-            this.importCount++;
-
-            const filteredElements = imports.elements.filter(e => {
-                const decorators = tsHelper.getCustomDecorators(this.checker.getTypeAtLocation(e), this.checker);
-                return !decorators.has(DecoratorKind.Extension) && !decorators.has(DecoratorKind.MetaExtension);
-            });
-
-            if (filteredElements.length === 0) {
-                return "";
-            }
-
-            filteredElements.forEach(element => {
-                const nameText = this.transpileIdentifier(element.name);
-                if (element.propertyName) {
-                    const propertyText = this.transpileIdentifier(element.propertyName);
-                    result += `local ${nameText} = ${fileImportTable}.${propertyText}\n`;
-                } else {
-                    result += `local ${nameText} = ${fileImportTable}.${nameText}\n`;
-                }
-            });
-
-            return result;
-        } else if (ts.isNamespaceImport(imports)) {
-            const resolvedImportPath = this.getImportPath(importPathWithoutQuotes);
-            return `local ${this.transpileIdentifier(imports.name)} = ${requireKeyword}(${resolvedImportPath})\n`;
-        } else {
-            throw TSTLErrors.UnsupportedImportType(imports);
-        }
+        // Handled in transform;
+        return "";
```

Comment:
Maybe we should throw an error if this function is actually still called?

Reply:
I think we can remove that function completely. Since all ImportDeclarations get replaced, not sure why I didn't do that already.